programs/config/src/lib.rs

```diff
@@ -7,16 +7,14 @@ pub mod config_processor;
 )]
 pub mod date_instruction;
 
-pub use solana_sdk::config::program::id;
+pub use solana_sdk_ids::config::id;
 #[allow(deprecated)]
-use solana_sdk::stake::config::Config as StakeConfig;
+use solana_stake_interface::config::Config as StakeConfig;
```

Comment:
Is this `#[allow(deprecated)]` still relevant?

Reply:
yes, the struct it imports is still deprecated